src/DurableTask.Netherite/Scaling/AzureBlobLoadPublisher.cs

```diff
@@ -50,44 +50,96 @@ Task UploadPartitionInfo(uint partitionId, PartitionLoadInfo loadInfo)
             {
                 var blobDirectory = this.blobContainer.GetDirectoryReference($"p{partitionId:D2}");
                 var blob = blobDirectory.GetBlockBlobReference("loadinfo.json");
-                var json = JsonConvert.SerializeObject(loadInfo, Formatting.Indented);
+                var json = JsonConvert.SerializeObject(loadInfo, Formatting.Indented, serializerSettings);
                 return blob.UploadTextAsync(json, cancellationToken);
             }
 
             List<Task> tasks = info.Select(kvp => UploadPartitionInfo(kvp.Key, kvp.Value)).ToList();
-            return Task.WhenAll(tasks);           
+            return Task.WhenAll(tasks);
```

Comment:
yes, I forgot about that. Will revise.

Reply:
I don't think I need to use `ConfigureAwait` on these lines since I am not using an await but returning the Task. Still, I get your point. There are other instances.